lightning/src/ln/channelmonitor.rs

```diff
@@ -823,13 +842,10 @@ impl<ChanSigner: ChannelKeys> PartialEq for ChannelMonitor<ChanSigner> {
 			self.funding_info != other.funding_info ||
 			self.current_remote_commitment_txid != other.current_remote_commitment_txid ||
 			self.prev_remote_commitment_txid != other.prev_remote_commitment_txid ||
-			self.their_htlc_base_key != other.their_htlc_base_key ||
```

Comment:
Please add the new fields?

Reply:
Damn I new I forgot it at some point but never committed... Corrected